extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1409,7 +1409,7 @@ public static function dplNumParserFunction(&$parser, $text='') {
 		return $num;
     } 
 
-    public static function dplVarParserFunction(&$parser, $cmd) {
+    public static function dplVarParserFunction($parser, $cmd) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)